### ISSUE TITLE
Output difference between Pure Nuke Implementation and Nuke Blink Node.

### ISSUE THREAD

**Issue by KelSolaar** [colour-science/aces-vwg-gamut-mapping-2020#5]
@jedypod, @nick-shaw :
While implementing the Python version, I just noticed a difference between the Pure Nuke Implementation and Nuke Blink Node for colours that are all negatives:

![image](https://user-images.githubusercontent.com/99779/89629158-792e0b00-d8f1-11ea-8e5c-64c2c9355756.png)

It has something to do with the `toe` node that is disabled according to `shd_rolloff`, the Blink Node seems to match my loose Python pot so far.

**Comment by nick-shaw** [comment]
Good catch @KelSolaar. It seems that when all three channels are negative, the Blink result is `r=g=b=max(rgb)`, whereas with pure Nuke the lower two are compressed toward the max value. The latter seems the more desirable result, but the Blink one will probably be what happens with all the others. I'll investigate.

**Comment by nick-shaw** [comment]
It appears that because when all three channels are negative, `max(rgb)` is negative, the [`ach_shd` calculation](https://github.com/colour-science/aces-vwg-gamut-mapping-2020/blob/6b5d9b313564ad3a7668ffe2ce9bf8f908248990/model/GamutCompress.blink#L206) fails because `(1.0-ach)<(1.0-shd_rolloff)` is false if `ach` is negative. Therefore instead of a passthru, the roll-off calculation is used which includes a division by `shd_rolloff` and thus produces an `inf` if `shd_rolloff` is zero.

At least that seems to me to be what is happening.

**Comment by nick-shaw** [comment]
When bypassing the shadow roll-off calculation in Blink, the results still don't match. This turns out to be because Nuke's `divide` merge mode is not a simple mathematical divide. It seems to be a pass-thru if the denominator is negative.

Nuke's merge mode description says:
```
divide A/B, 0 if A<0 and B<0
```
But it would appear to be:
```
divide A/B, A if B<0
```
